prizes.md

```diff
@@ -17,10 +17,34 @@ Best piece of content published about Impact Framework, an article, tutorial, vi
 ### ✨ Best Contribution to the Framework
 Best contribution to the Impact Framework tool itself. Adding a new feature or enhancing an existing one.
 
-_____
-We are also keen to broaden the opportunities for those who are studying:
+---
+
+_We are also keen to broaden the opportunities for those who are studying:_
 ### 🎓 Best Undergraduate
 Specific prize for Undergraduate Students, drawn from across the main categories.
 
+---
+
+Sponsoring organizations can have teams participate in the hackathon. To ensure fairness and maintain the integrity of this event, judges will be asked to recuse themselves from scoring where their organization is concerned.
+
+---
+
+### The Judging Process
+Carbon Hack submissions will close on Friday 15th March at 23.59 GMT (UK Time). After that time, there will be a shortlisting and judging process.
+
+#### Submitting your Project
```

Comment:
Leave out the judging process, we'll add it in as a separate file once we have decided more concretely areas in that realm. I've left feedback in participating, how to submit should be detailed in the participating file.